### PR TITLE
[FEATURE] Aggregate marm model

### PR DESCRIPTION
**Pull request by adamocarolli**
### Description
Add marm model aggregation.

### Changes
- Add aggregated marm model JSON to static file server
- Replace marm parsed graph representation with pre-aggregated representation 
- Lower edge opacity on larger models

### Considerations
- Here we perform offline aggregation of the marm model instead of modifying the Python code to Javascript and running in browser. Either option is valid, however, since the Python code had been unstable/changing I didn't want to add an extra dependency in translating it over and over again if we made another change.
- Scalable flow graphs is something that needs additional time investment. I think that we should start moving the renderer over to grafer for more performant large graph handling, however, we'll need to add an expand/collapse feature and a stable camera during expansion. The other major bottleneck to these visualizations is that the ELK layout algorithm we are using is not very performant taking roughly 2s to compute over this model on expansion. We may need to re-write this from scratch and see if we can improve on performance via localized layout recalculations/caching. 

### Testing
- No special instructions.

### Artifacts:
Screen Shot 2021-08-11 at 12 40 12 PM
Screen Shot 2021-08-11 at 12 41 31 PM

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
Looks neat!

File: client/src/graphs/svg/renderers/EpiRenderer.ts [hunk 1]
```diff
@@ -48,6 +49,10 @@ export default class EpiRenderer extends SVGRenderer {
     } else return DEFAULT_STYLE.node.strokeWidth;
   }
 
+  static calcEdgeOpacity (edgeSelection: d3.Selection<any, any, any, any>): number {
+    return edgeSelection.size() >= 100 ? 0.1 : DEFAULT_STYLE.edge.opacity;
```
[the remark this answers is not in the document]
Comment (RosaRomeroGomez) [reply client/src/graphs/svg/renderers/EpiRenderer.ts:53]: I actually think that maybe opacity 0.1 is too much, what about 0.3. Sorry about that.

**Review by RosaRomeroGomez — APPROVED** [review]
Thanks!

File: client/src/graphs/svg/renderers/EpiRenderer.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamocarolli) [reply client/src/graphs/svg/renderers/EpiRenderer.ts]: Both done, thanks!

**Pull request merged**